instrumentation/common-api/src/test/java/io/opentelemetry/android/instrumentation/common/ScreenNameExtractorTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.common
+
+import android.app.Activity
+import android.app.Fragment
+import io.opentelemetry.android.instrumentation.annotations.RumScreenName
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class ScreenNameExtractorTest {
+    @Test
+    fun testActivity() {
+        val activity = Activity()
+        val name = ScreenNameExtractor.DEFAULT.extract(activity)
+        Assertions.assertEquals("Activity", name)
```

Comment:
Please use static imports for these, or ever better, migrate to the fluent assertj assertions: `assertThat(name).isEqualTo("Activity")`. 

I know you didn't write these and migrating to kotlin was just part of the cleanup, but it would help here. Thanks!